testutil/compose/docker-compose.template

```diff
@@ -68,19 +68,17 @@ services:
 
   prometheus:
     image: prom/prometheus:latest
-    {{if .MonitoringPorts}}ports:
```

Comment:
Instead of not binding ports, rather just don't run monitoring at all (since not used if bound). Also saves some resources since less containers.

Reply:
actually, we do not monitoring since we rely on alerts for health checks